packages/v4/src/content/accessibility/progress-stepper/progress-stepper.md

```diff
@@ -0,0 +1,19 @@
+---
+id: Progress stepper
+section: components
+---
+
+A **progress stepper** is used to provide information to the user about the state of their current progress on a multistep interaction.
+
+**Keyboard users** should be able to use **Tab** and **Shift + Tab** navigate to and between progress steps with popovers. Keyboard users should 
+be able to open and close the popover on a progress step using **Enter** or **Space**.
+
+**Screen reader users** should be able to discern the label, status, and optional description of every progress step. 
```

Comment:
For NVDA/JAWS it might just be called "elements list"? Based on this screen shot of keyboard shortcuts from Deque, the task in the second column aligns with what the rotor does in VO (the first two columns are the same for their pages on NVDA and JAWS)

![NVDA elements list shortcut](https://user-images.githubusercontent.com/70952936/161761463-59709af9-8187-42ea-b21a-d282582006ee.png)

That said I think I'd agree in maybe explicitly stating in what way the info should be discernible via screen reader, re: screen reader/VO focus vs rotor/element list. This might involve doing a run-through of current a11y docs as well, but could be easier at this stage.